lib/ansible/modules/files/blockinfile.py

```diff
@@ -126,6 +126,12 @@
           address 192.0.2.23
           netmask 255.255.255.0
 
+– name: insert/update configuration using a local file
+  blockinfile:
+    block: "{{ lookup(‘file’, ‘./local/ssh_config’) }}"
```

Comment:
I think you've got an dodgy quote in that line, I think you need:
`block: "{{ lookup(‘file', './local/ssh_config') }}"`

That should fix the Shippable error